automation/test.sh

```diff
@@ -303,7 +320,9 @@ fi
 
 # Set KUBEVIRT_E2E_FOCUS and KUBEVIRT_E2E_SKIP only if both of them are not already set
 if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
-  if [[ $TARGET =~ windows.* ]]; then
+  if [[ $TARGET =~ windows_sysprep.* ]]; then
+    export KUBEVIRT_E2E_FOCUS=sysprep
```

Comment:
I think it would be better to refer to Windows in the focus, something like `Windows-sysprep`, wdyt?

Reply:
I would like to keep this as sysprep, sysprep is always windows and this way I do not need to change the existing windows tests description or focus.